src/popup/Popup.tsx

```diff
@@ -34,12 +38,46 @@ export default function Popup(): JSX.Element {
     <input type="checkbox" name="emotesCheckbox" onChange={changeEmoteStatus} />
   );
 
+  /* Theme Hook + Conditonal Rendering */
+
+  const changeThemeStatus = useCallback(() => {
+    const currentThemeStatus = !darkThemeEnabled;
+    setDarkTheme((darkThemeEnabled) => !darkThemeEnabled);
+    const message: string = currentThemeStatus
+      ? 'enableDarkTheme'
+      : 'disableDarkTheme';
+    chrome.tabs.query({ active: true, currentWindow: true }, function (tabs) {
+      chrome.tabs.sendMessage(tabs[0].id, {
+        darkThemeEnabled: currentThemeStatus,
+        message: message
+      });
+    });
+  }, [darkThemeEnabled]);
+
+  const themeButton = darkThemeEnabled ? (
+    <input
```

Comment:
I don't think we need to conditionally render the checkbox. Idk exactly how to make it work so we don't have to, so I'll say this is fine for now but maybe later I will go in and clean up this React code